### PR TITLE
Update 8tracks as hard

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

**Comment by WebSnke** [comment]
> Were you able to gain contact with them? 

No, I didn't contact them but the information I added is part of their privacy policy.

**Comment by tupaschoal** [comment]
But were you able to delete an account with them @WebSnke ?

**Comment by WebSnke** [comment]
> But were you able to delete an account with them @WebSnke ?

I have never had a account on 8tracks.

But this is what 8tracks privacy policy says: 

"You may view, update, correct, or delete your user information and preferences by contacting us at [privacy@8tracks.com](mailto:privacy@8tracks.com)."

**Comment by tupaschoal** [comment]
Hmm, that puts us in a weird position. We typically ask people to submit proven instructions. @SvicidalBug tried to delete and couldn't, hence the impossible, but if it proved to work now, then we can change. Sadly we cannot trust on privacy policies alone, because we have seen multiple times places where it failed.

**Comment by WebSnke** [comment]
> Hmm, that puts us in a weird position. We typically ask people to submit proven instructions. @SvicidalBug tried to delete and couldn't, hence the impossible, but if it proved to work now, then we can change. Sadly we cannot trust on privacy policies alone, because we have seen multiple times places where it failed.

Ok, I will try to create an account and delete it soon. It would have to work as I am in the EU, fortunately.

**Comment by WebSnke** [comment]
> > Hmm, that puts us in a weird position. We typically ask people to submit proven instructions. @SvicidalBug tried to delete and couldn't, hence the impossible, but if it proved to work now, then we can change. Sadly we cannot trust on privacy policies alone, because we have seen multiple times places where it failed.
> 
> Ok, I will try to create an account and delete it soon. It would have to work as I am in the EU, fortunately.

There is now a delete button. I will try to use that one.
image

**Review by tupaschoal — APPROVED** [review]
Thanks for investigating!

File: _data/sites.json [hunk 1]
```diff
@@ -187,10 +187,10 @@
     },
 
     {
-        "name": "8tracks",
-        "url": "https://8tracks.com/",
-        "difficulty": "impossible",
-        "notes": "There exists an option to delete your account in your account settings, but it's not currently functional.",
+        "name": "8tracks"
```
Comment (tupaschoal) [inline _data/sites.json:190]:
```suggestion
        "name": "8tracks",
```

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**